src/Test/Desktop/EFCodePOCOModels/Northwind.edmx

```diff
@@ -0,0 +1,158 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx Version="3.0" xmlns:edmx="http://schemas.microsoft.com/ado/2009/11/edmx">
```

Comment:
I dont think we need to add any edmx file, the singel code first model in EfCoreModels project should be enough